app/src/main/java/com/example/peterchu/watplanner/Networking/ApiInterface.java

```diff
@@ -15,28 +15,21 @@
 
 public interface ApiInterface {
     @GET("terms/{termId}/courses.json")
-    Call<CourseResponse> getCourses(@Path("termId") String termId, @Query("key") String apiKey);
+    Call<CourseResponse> getCourses(
+            @Path("termId") String termId,
+            @Query("key") String apiKey);
 
-    @GET("terms/{termId}/{subject}/{catalogNumber}/schedule.json")
+    @GET("courses/{subject}/{catalogNumber}/schedule.json")
     Call<CourseScheduleResponse> getCourseSchedule(
-            @Path("termId") String termId,
             @Path("subject") String subject,
             @Path("catalogNumber") String catalogNumber,
             @Query("key") String apiKey
     );
 
-    @GET("terms/{termId}/{subject}/schedule.json")
```

Comment:
Previously we were just retrieving all course schedules under a certain subject (e.g. ECE) and storing that into the DB. Instead, we just retrieve the single course schedule when we need it: Viewing course details.

Down the the line, I'm thinking if we save a course we also store it in the DB